### DISCUSSION TITLE
Controlling Lulu via Terminal

### DISCUSSION THREAD

**Discussion by OverjoyedFrankenstein** [objective-see/LuLu#614] [Ideas]
I've looked around a bit to see if I could find any info. I didn't find anything. Forgive me if this is something I'm unaware of.

It would be great for some of my automations to be able to control Lulu via Terminal. A few useful examples are below:
lulu.app -enable (to enable firewall)
lulu.app -disable (to disable firewall)
lulu.app -blockall domain.url.com (block any traffic to this url)
lulu.app -blockapp "appcleaner" (prevent any app with "appcleaner" in the name to communicate in or out)

**Comment by objective-see** [comment]
I didn't add this functionality, as I was worried malware could abuse it to, for example, disable the firewall :\ 
(Apple does a great job "protecting" UIs, so malware can't do it via interacting w/ the UI, where this functionality is currently (only) exposed).

**Comment by OverjoyedFrankenstein** [comment]
Thanks! Currently I've got a cron job that uses the killall command to shut off Lulu, then the open command to relaunch it after the computer does what it needs unrestricted online. These things will work fine for my usage. Keep up the good work!
